development/secrets-rotator/terraform/environments/prod/variables.tf

```
@@ -3,6 +3,11 @@ variable "project_id" {
   description = "Google Cloud project ID to deploy the secret rotator application to."
 }
 
+variable "project_workloads_id" {
```

```suggestion
variable "workloads_project_id" {
```
`project_workloads_id` sounds for me more like ids of workloads of a project not an id of a project for workloads. References to the variable should be aligned.

---

Done